test/addons/rook-cluster/fetch

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
```

Comment:
Please wrap long lines in the commit message. Using width of 72 characters for commit messages helps to get more readable text.